catalog.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>HTML Academy: Девайс</title>
+  <link href="device/css/css.css" rel="stylesheet">
```

Comment:
In task it said, that is should be names `style.css`:
```
Разметка всех страниц и общий файл style.css уже были созданы в двух предыдущих разделах, стилевой файл также был подключён ко всем страницам.
```
Let's rename it to `style.css`